README.md

```diff
@@ -0,0 +1,36 @@
+# AWS SOCI Index Builder Documentation
```

Comment:
Can we add link to the rendered html from here: https://awslabs.github.io/cfn-ecr-aws-soci-index-builder/